package.json

```diff
@@ -39,6 +39,7 @@
     "@babel/preset-env": "7.12.7",
     "@babel/preset-react": "7.12.10",
     "@babel/preset-typescript": "7.12.7",
+    "@csstools/postcss-sass": "4.0.0",
```

Comment:
This shouldn't be part of this diff.

Reply:
Yeah that was because the target branch changed and I didn't rebase; it's gone now.